libindy/src/commands/metrics.rs

```diff
@@ -55,26 +56,47 @@ impl MetricsCommandExecutor {
     }
 
     fn append_threapool_metrics(&self, metrics_map: &mut Map<String, Value>) {
+        struct MetricsTags {
+            stage: String,
+        }
         let tp_instance = crate::commands::THREADPOOL.lock().unwrap();
-        metrics_map.insert(String::from(THREADPOOL_ACTIVE_COUNT),
-                           Value::from(tp_instance.active_count()));
-        metrics_map.insert(String::from(THREADPOOL_QUEUED_COUNT),
-                           Value::from(tp_instance.queued_count()));
-        metrics_map.insert(String::from(THREADPOOL_MAX_COUNT),
-                           Value::from(tp_instance.max_count()));
-        metrics_map.insert(String::from(THREADPOOL_PANIC_COUNT),
-                           Value::from(tp_instance.panic_count()));
+        let metrics_map = checkMetricsStructure(metrics_map);
```

Comment:
1. I would prefer to stick to snake_case for function names to be consistent with the rest of codebase
2. Function that mutates it's arguments shouldn't be named like `check_something`. I would propose `ensure_metrics_contain_<yadayada>`.
3. I checked contents of this function - and it is commented out. How does this code compile at all?
4. This function ensures that there are command-metrics inside of map - is it really needed, and if yes - why is this being done in function that is supposed to add threadpool-related metrics?